### PR TITLE
Merge Sort

### PR DESCRIPTION
**Pull request by Ronak55**
Implemented merge sort algorithm in both cpp and Javascript

### CODE REVIEW

**Comment by ayan-biswas0412** [comment]
> Implemented merge sort algorithm in both cpp and python

I think you have done Javascript and cpp right ?

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
Please add proper heading for each of the languages you have implemented

**Review by Ronak55 — COMMENTED** [review]
My bad, i just wrote python by mistake i did it on js and cpp

File: docs/sorting/merge_sort.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        mergeSort(arr, l, m);
+        mergeSort(arr, m + 1, r);
+
+        merge(arr, l, m, r);
+    }
+}
+
+
+```
+
```
Comment (ayan-biswas0412) [inline docs/sorting/merge_sort.md:56]:
```suggestion
### JavaScript Implementation
```

**Pull request merged**